content/docs/visualization/promdash.md

```diff
@@ -129,6 +129,19 @@ interpolated version of the given format string. To reference specific label
 values in the format string, use double curly braces: `{{label-name}}`. For
 example: `{{host}} - cluster {{cluster}}`.
 
+### Legend format filters
+The aforementioned `Legend format` builder also supports filters for the
+labels. The format is `{{label | filter}}`. The following filters are available:
+
+- `toPercent`
```

Comment:
Could you mention for each of these what input they expect, and what the output is? For some, it's not really clear just from the name.